### PR TITLE
config update

### PR DESCRIPTION
**Pull request by thecalcc**
STTNHUB-289

### CODE REVIEW

**Comment by thecalcc** [comment]
@petrjasek can you check backend tests? Also not sure how we should update the webpack changes causing frontend to fail

**Comment by petrjasek** [comment]
you can merge develop there I fixed it

**Review by petrjasek — APPROVED** [review]

File: server/settings.py [hunk 1]
```diff
@@ -217,6 +220,5 @@
 ]["filter"] = ["lowercase"]
 
 AGENDA_HIDE_COVERAGE_ASSIGNEES = True
-AGENDA_DEFAULT_FILTER_HIDE_PLANNING = True
```
Comment (petrjasek) [inline server/settings.py]: why removing this one?

Comment (thecalcc) [reply server/settings.py]: my bad

**Pull request merged**